integration-tests/pkg/collector/introspection.go

```diff
@@ -0,0 +1,26 @@
+package collector
+
+import (
+	"fmt"
+	"io"
+	"net/http"
+
+	"github.com/stackrox/collector/integration-tests/pkg/log"
+)
+
+func (k *K8sCollectorManager) IntrospectionQuery(endpoint string) ([]byte, error) {
+	uri := fmt.Sprintf("http://%s:8080%s", k.IP(), endpoint)
+	body := []byte{}
```

Comment:
Unneeded declaration of body.

Reply:
When there is an error you can return nil instead of body. That is the way that errors are usually handled in the stackrox repo.

Reply:
I didn't know `nil` could be used with arrays and maps TIL, thanks!